### ISSUE TITLE
New submissions for Tue,  2 May 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#5]
# New submissions for Tue,  2 May 23
Update Star Formation & Molecular Cloud papers at 2am UTC (10am Beijing time) every weekday, forked from zhuhu00/Paper-Daily-Notice. 
2023-05-02
### Recent Progress in Modelling the Macro- and Micro-Physics of Radio Jet  Feedback in Galaxy Clusters
 - **Authors:** Martin A. Bourne, Hsiang-Yi Karen Yang
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2305.00019
 - **Pdf link:** https://arxiv.org/pdf/2305.00019
 - **Abstract**
 Radio jets and the lobes they inflate are common in cool-core clusters and are expected to play a critical role in regulating the heating and cooling of the intracluster medium (ICM). This is an inherently multi-scale problem, and much effort has been made to understand the processes governing the inflation of lobes and their impact on the cluster, as well as the impact of the environment on the jet-ICM interaction, on both macro- and microphysical scales. Developments of new numerical techniques and improving computational resources have seen simulations of jet feedback in galaxy clusters become ever more sophisticated. This ranges from modelling ICM plasma physics processes such as the effects of magnetic fields, cosmic rays and viscosity to including jet feedback in cosmologically evolved cluster environments in which the ICM thermal and dynamic properties are shaped by large-scale structure formation. In this review, we discuss the progress made over the last ~decade in capturing both the macro- and microphysical processes in numerical simulations, highlighting both the current state of the field as well as open questions and potential ways in which these questions can be addressed in the future.
### Kinematics and stability of high-mass protostellar disk candidates at  sub-arcsecond resolution -- Insights from the IRAM NOEMA large program CORE
 - **Authors:** Aida Ahmadi, H. Beuther, F. Bosco, C. Gieser, S. Suri, J. C. Mottram, R. Kuiper, Th. Henning, Á. Sánchez-Monge, H. Linz, R. E. Pudritz, D. Semenov, J. M. Winters, T. Möller, M. T. Beltrán, T. Csengeri, R. Galván-Madrid, K. G. Johnston, E. Keto, P. D. Klaassen, S. Leurini, S. N. Longmore, S. L. Lumsden, L. T. Maud, L. Moscadelli, A. Palau, T. Peters, S. E. Ragan, J. S. Urquhart, Q. Zhang, H. Zinnecker
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2305.00020
 - **Pdf link:** https://arxiv.org/pdf/2305.00020
 - **Abstract**
 The fragmentation mode of high-mass molecular clumps and the accretion processes that form the most massive stars ($M\gtrsim 8M_\odot$) are still not well understood. To this end, we have undertaken a large observational program (CORE) making use of interferometric observations from the Northern Extended Millimetre Array (NOEMA) for a sample of 20 luminous ($L>10^4L_\odot$) protostellar objects in the 1.37 mm wavelength regime in both continuum and line emission, reaching $\sim$0.4" resolution (800 au at 2 kpc). Using the dense gas tracer CH$_3$CN, we find velocity gradients across 13 cores perpendicular to the directions of bipolar molecular outflows, making them excellent disk candidates. Specific angular momentum ($j$) radial profiles are on average $\sim10^{-3}$ km /s pc and follow $j \propto r^{1.7}$, consistent with a poorly resolved rotating and infalling envelope/disk model. Fitting the velocity profiles with a Keplerian model, we find protostellar masses in the range of $\sim 10-25$ $M_\odot$. Modelling the level population of CH$_3$CN lines, we present temperature maps and find median gas temperatures in the range $70-210$ K. We create Toomre $Q$ maps to study the stability of the disks and find almost all (11 of 13) disk candidates to be prone to fragmentation due to gravitational instabilities at the scales probed by our observations. In particular, disks with masses greater than $\sim10-20\%$ of the mass of their host (proto)stars are Toomre unstable, and more luminous protostellar objects tend to have disks that are more massive and hence more prone to fragmentation. Our finings show that most disks around high-mass protostars are prone to disk fragmentation early in their formation due to their high disk to stellar mass ratio. This impacts the accretion evolution of high-mass protostars which will have significant implications for the formation of the most massive stars.
### Machine Learning Uncovers the Universe's Hidden Gems: A Comprehensive  Catalogue of CIV Absorption Lines in SDSS DR12
 - **Authors:** Reza Monadi, Ming-Feng Ho, Kathy L. Cooksey, Simeon Bird
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2305.00023
 - **Pdf link:** https://arxiv.org/pdf/2305.00023
 - **Abstract**
 We assemble the largest CIV absorption line catalogue to date, leveraging machine learning, specifically Gaussian processes, to remove the need for visual inspection for detecting CIV absorbers. The catalogue contains probabilities classifying the reliability of the absorption system within a quasar spectrum. Our training set was a sub-sample of DR7 spectra that had no detectable CIV absorption in a large visually inspected catalogue. We used Bayesian model selection to decide between our continuum model and our absorption-line models. Using a random hold-out sample of 1301 spectra from all of the 26,030 investigated spectra in DR7 CIV catalogue, we validated our pipeline and obtained an 87% classification performance score. We found good purity and completeness values, both ~80%, when a probability of ~95% is used as the threshold. Our pipeline obtained similar CIV redshifts and rest equivalent widths to our training set. Applying our algorithm to 185,425 selected quasar spectra from SDSS DR12, we produce a catalogue of 113,775 CIV doublets with at least 95% confidence. Our catalogue provides maximum a posteriori values and credible intervals for CIV redshift, column density, and Doppler velocity dispersion. We detect CIV absorption systems with a redshift range of 1.37 $\!-\!$ 5.1, including 33 systems with a redshift larger than 5 and 549 absorbers systems with a rest equivalent width greater than 2 A at more than 95% confidence. Our catalogue can be used to investigate the physical properties of the circumgalactic and intergalactic media.
### Detection of monothioformic acid towards the solar-type protostar IRAS  16293-2422
 - **Authors:** Arijit Manna, Sabyasachi Pal
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2305.00480
 - **Pdf link:** https://arxiv.org/pdf/2305.00480
 - **Abstract**
 In the interstellar medium (ISM), the complex organic molecules that contain the thiol group ($-$SH) play an important role in the polymerization of amino acids. We look for SH-bearing molecules in the chemically rich solar-type protostar IRAS 16293-2422. After the extensive spectral analysis using the local thermodynamic equilibrium (LTE) model, we have detected the rotational emission lines of trans-isomer monothioformic acid (t-HC(O)SH) towards the IRAS 16293 B using the Atacama Large Millimeter/Submillimeter Array (ALMA). We did not observe any evidence of cis-isomer monothioformic acid (c-HC(O)SH) towards the IRAS 16293 B. The column density of t-HC(O)SH towards the IRAS 16293 B was (1.02$\pm$0.6)$\times$10$^{15}$ cm$^{-2}$ with an excitation temperature of 125$\pm$15 K. The fractional abundance of t-HC(O)SH with respect to H$_{2}$ towards the IRAS 16293 B is 8.50$\times$10$^{-11}$. The column density ratio of t-HC(O)SH/CH$_{3}$SH towards the IRAS 16293 B is 0.185. We compare our estimated abundance of t-HC(O)SH towards the IRAS 16293 B with the abundance of t-HC(O)SH towards the galactic center quiescent cloud G+0.693-0.027 and hot molecular core G31.41+0.31. After the comparison, we found that the abundance of t-HC(O)SH towards the IRAS 16293 B is several times of magnitude lower than G+0.693-0.027 and G31.41+0.31. We also discuss the possible formation mechanism of t-HC(O)SH in the ISM.
### Joint Modelling of Dust Scattering and Thermal Emission: The Spider  Complex
 - **Authors:** Jielai Zhang, Peter G Martin, Ryan Cloutier, Natalie Price-Jones, Roberto Abraham, Pieter van Dokkum, Allison Merritt
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2305.00756
 - **Pdf link:** https://arxiv.org/pdf/2305.00756
 - **Abstract**
 Observations across the electromagnetic spectrum of radiative processes involving interstellar dust -- emission, extinction, and scattering -- are used to constrain the parameters of dust models and more directly to aid in foreground removal of dust for extragalactic and cosmology observations. The more complementary observations, the better. Here, we quantify the relationship between scattered light and thermal emission from dust in a diffuse (cirrus) intermediate latitude cloud, Spider, using data from the Dragonfly Telephoto Array and the Herschel Space Observatory. A challenge for optical observations of faint cirrus is accurate removal of a contaminating spatially varying sky background. We present a technique to analyse two images of the same cirrus field concurrently, correlating pixel values to capture the relationship and simultaneously fitting the sky background as a complex non-correlating additive component. For the Spider, we measure a $g-r$ color of 0.644$\pm 0.024$ and a visible wavelength to 250 $\mu$m intensity ratio of $10^{-3} \times (0.855 \pm0.025)$ and $10^{-3} \times (1.55\pm0.08)$ for $g$ and $r$-band respectively. We show how to use any dust model that matches the thermal dust emission to predict an upper limit to the amount of scattered light. The actual brightness of the cirrus will be fainter than this limit because of anisotropic scattering by the dust combined with anisotropy of the incident interstellar radiation field (ISRF). Using models of dust and the ISRF in the literature we illustrate that the predicted brightness is indeed lower, though not as faint as the observations indicate.
### Small Planets Around Cool Dwarfs: Enhanced Formation Efficiency of  Super-Earths around M dwarfs
 - **Authors:** Yayaati Chachan, Eve J. Lee
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2305.00803
 - **Pdf link:** https://arxiv.org/pdf/2305.00803
 - **Abstract**
 Current measurements of planet population as a function of stellar mass show three seemingly contradictory signatures: close-in super-Earths are more prevalent around M dwarfs than FGK dwarfs; inner super-Earths are correlated with outer giants; and outer giants are less common around M dwarfs than FGK dwarfs. Here, we build a simple framework that combines the theory of pebble accretion with the measurements of dust masses in protoplanetary disks to reconcile all three observations. First, we show that cooler stars are more efficient at converting pebbles into planetary cores at short orbital periods. Second, when disks are massive enough to nucleate a heavy core at 5 AU, more than enough dust can drift in to assemble inner planets, establishing the correlation between inner planets and outer giants. Finally, while stars of varying masses are similarly capable of converting pebbles into cores at long orbital periods, hotter stars are much more likely to harbor more massive dust disks so that the giant planet occurrence rate rises around hotter stars. Our results are valid over a wide range of parameter space for a disk accretion rate that follows $\dot{M}_\star \sim 10^{-8}\,M_\odot\,{\rm yr}^{-1}(M_\star/M_\odot)^2$. We predict a decline in mini-Neptune population (but not necessarily terrestrial planets) around stars lighter than $\sim 0.3-0.5 \, M_\odot$. Cold giants ($\gtrsim$5 AU), if they exist, should remain correlated with inner planets even around lower mass stars.
